app/(main)/login/page.test.tsx

```diff
@@ -33,10 +41,17 @@ jest.mock('../../../context/AuthContextProvider', () => ({
 }));
 
 describe('Login', () => {
+  const setIsLoading = jest.fn();
+  const useStateMock = (init: any): [unknown, Dispatch<unknown>] => [
+    init,
+    setState,
+  ];
   beforeEach(() => {
     jest.clearAllMocks();
+    jest.spyOn(React, 'useState').mockImplementation(useStateMock);
```

Comment:
Does this not return an error in your local machine? I tried using this syntax for my loadingSpinner test, but it kept returning an error for me.

Reply:
@choir27 it does; and it did before. I just haven't been able to get back to this one :P

Reply:
@choir27 fixed